core/src/main/java/nu/yona/server/subscriptions/service/BuddyDisconnectMessageDTO.java

```diff
@@ -120,7 +120,7 @@ public MessageActionDTO handleAction(UserDTO actingUser, Message messageEntity,
 		private MessageActionDTO handleAction_Process(UserDTO actingUser, BuddyDisconnectMessage messageEntity,
 				MessageActionDTO requestPayload)
 		{
-			buddyService.removeBuddyAfterBuddyRemovedConnection(actingUser.getID(), messageEntity.getUserID());
+			buddyService.removeBuddyAfterBuddyRemovedConnection(actingUser.getID(), messageEntity.getSenderUserID());
```

Comment:
Hmm, on further thought this is maybe not the best. The user may not be retrieved but his messages may be...